examples/seismic/model.py

```diff
@@ -40,9 +40,10 @@ def initialize_damp(damp, nbl, spacing, abc_type="damp", fs=False):
             val = -val if abc_type == "mask" else val
             eqs += [Inc(damp.subs({d: dim_l}), val/d.spacing)]
         # right
+        dampcoeff = 1.5 * np.log(1.0 / 0.001) / (nbr)
         dim_r = SubDimension.right(name='abc_%s_r' % d.name, parent=d,
-                                   thickness=nbl)
-        pos = Abs((nbl - (d.symbolic_max - dim_r) + 1) / float(nbl))
+                                   thickness=nbr)
```

Comment:
should we rather call these variables `nbll` and `nblr` ?

Reply:
I think nbl/nbr is fine, what we use in builtins as well.